### PR TITLE
create basic screens and navigation

### LINKED ISSUE #9: Code Splash screen
As a developer
I want to see the splash screen on load for 2 sec

**Requirements**
1. Splash screen looks like "Figma" prototype with logo and description
2. Visible for 2 sec
3. Colors are consistent with "Figma" prototype

### LINKED ISSUE #10: Code Home screen
As a developer
I want to see the home screen after splash screen is loaded

**Requirements**
1. Home screen looks like "Figma" prototype (no multiple pictures yet)
2. Loads after Splash screen is loaded (2 sec)
3. Colors are consistent with "Figma" prototype

### LINKED ISSUE #11: Code Select screen
As a developer
I want to see the select screen when clicking on "In the Park" picture

**Requirements**
1. Select screen looks like "Figma" prototype without animation
2. Colors are consistent with "Figma" prototype
3. Button is not active yet

### PR DESCRIPTION
**Pull request by darowsluk**
fixes #9
fixes #10
fixes #11

### CODE REVIEW

File: lib/views/select_screen.dart [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                          .bodyText1
+                          ?.copyWith(color: AppColors.primaryWhite),
+                    ),
+                  ),
+                  Padding(padding: EdgeInsets.all(60)),
+                  TextButton(
+                    child: Text("Zaczynamy"),
+                    onPressed: () {
+                      print('Pressed');
+                    },
```
Comment (knutm92) [inline lib/views/select_screen.dart:81]: Looking at Figma, some roundiness could be introduced here, maybe something along these lines as another parameter in `TextButton`:
`                    style: ButtonStyle(
                        shape:
                            MaterialStateProperty.all<RoundedRectangleBorder>(
                                RoundedRectangleBorder(
                      borderRadius: BorderRadius.circular(18.0),
                    )))`

**Review by knutm92 — CHANGES_REQUESTED** [review]
Great to see something tangible! 


Writing a Splash-screen related comment here, as I couldn't get the hang where it resides in the code :)

#9 :
* Text lines with description can be added beneath the logo to make it consistent with Figma Prototype.

Other than these three bits, the code seems good 👍

File: lib/views/select_screen.dart (hunk 1 above)
Comment (darowsluk) [reply lib/views/select_screen.dart:81]: Fixed with radius 20 in agreement with Figma.

File: lib/views/home_screen.dart [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            Tab(
+                icon: Icon(
+              CupertinoIcons.question,
+              color: AppColors.primaryNormal,
+            )),
+          ],
+        ),
+        body: TabBarView(
+          children: [
+            Column(
```
[the remark this answers is not in the document]
Comment (darowsluk) [reply lib/views/home_screen.dart:74]: fixed with scrollable view

**Comment by darowsluk** [comment]
> Great to see something tangible!
> 
> Writing a Splash-screen related comment here, as I couldn't get the hang where it resides in the code :)
> 
> #9 :
> 
> * Text lines with description can be added beneath the logo to make it consistent with Figma Prototype.
> 
> Other than these three bits, the code seems good 👍

I propose removing text from the splash screen in the requirements for now.
Creating splash screen with text is more involved and we still don't have a good name and a logo.

**Comment by knutm92** [comment]
> I propose removing text from the splash screen in the requirements for now.
Creating splash screen with text is more involved and we still don't have a good name and a logo.

Okay, this sounds reasonable👍Let's add splash screen text to the backlog then.

**Review by knutm92 — APPROVED** [review]
Changes look good to me, let's merge.

**Pull request merged**